1.illumination_correction/scripts/0.create_loaddata_csvs.py

```diff
@@ -98,6 +113,47 @@
 # In[5]:
 
 
+# Step 4: Initialize storage to track used files and find proper column order 
+br00_dataframes = {br_id: [] for br_id in br00_ids}
+used_files = set()  # Store filenames used in the process
+concat_files = []  # Track new concatenated CSV files
+
+# Load one BR00 starting CSV that will have the correct column order
+column_order = pd.read_csv(pathlib.Path(f"{output_csv_dir}/{list(br00_ids)[0]}_loaddata_original.csv"), nrows=0).columns.tolist()
+
+# Step 5: Add 'Metadata_Reimaged' column and group by BR00 ID
+for csv_file in csv_files:
+    filename = csv_file.stem
+    match = br00_pattern.search(filename)
+
+    if match:
+        br_id = match.group(1)
+
+        # Read the CSV file into a DataFrame
+        df = pd.read_csv(csv_file)
+
+        # Reorder DataFrame columns to match the correct column order
+        df = df[column_order]  # Ensure the columns are in the correct order
+
+        # Add 'Metadata_Reimaged' column based on filename
+        df['Metadata_Reimaged'] = 'Re-imaged' in filename
```

Comment:
fwiw, i prefer the `assign()` syntax for adding columns:

```python
is_reimaged_set = "Re-imaged" in filename
df = df.assign(Metadata_Reimaged = is_reimaged_set)
```

(just fyi, no need to change if working)

(although `df` isn't a great variable name 😂 )

Reply:
Fair about `df` 😆! I can update it to be more descriptive! 

I haven't heard of using assign but I will have to consider it next time! I will keep this for now since it is working :)